sample-node-config/aws/docker/docker-compose.yaml

```diff
@@ -0,0 +1,100 @@
+version: "3.9"
+
+services:
+  cardano-node:
+    image: inputoutput/cardano-node:latest
+    restart: always
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "100m"
+        max-file: "5"
+    volumes:
+      - .:/data
+      - ./devnet:/devnet
+    working_dir: /data
+    environment:
+      - CARDANO_SOCKET_PATH=/devnet/node.socket # used by cardano-node
+      - CARDANO_NODE_SOCKET_PATH=/devnet/node.socket # used by cardano-cli
+    command:
+      [ "run"
+      , "--config", "/devnet/cardano-node/config.json"
+      , "--topology", "/devnet/cardano-node/topology.json"
+      , "--database-path", "/devnet/db"
+      ]
+
+  hydra-node:
+    image: abailly/hydra-node:latest
```

Comment:
@abailly-iohk shall we just decide to go back to push `master` builds as `latest` and update our demo (more settled) setups to `0.6.0` and bump these versions upon release instead? I think that was the consensus last time, but it needs to be clear that we are trading efforts with this.

Reply:
I think so, yes. Perhaps worthy of a small ADR?

Reply:
(happy to contribute...)